roles/ngi-rnaseq/tasks/main.yml

```diff
@@ -2,16 +2,11 @@
 - name: Create NextFlow directory
   file: path="{{ nextflow_dest }}" state=directory mode=g+rwXs
 
-- name: Put NextFlow installation files in same directory
-  shell: "export NXF_HOME={{ nextflow_dest }}"
-
-- name: Add NextFlow installation files to env var NFX_HOME
-  lineinfile: dest="{{ ngi_pipeline_conf }}/{{ bash_env_script }}"
-              line="export NXF_HOME={{ nextflow_dest }}"
-              backup=no
-
 - name: Install NextFlow
   shell: "./nextflow"
+  environment:
```

Comment:
They are necessary for the './nextflow' command to function correctly; e.g. install files in the directory I want them to be placed in. This is implied by having them set in the 'Install NextFlow' task.